fe/lib/components/Questionnaire/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -17,22 +17,33 @@ import {
   QuestionnaireCreateInput,
   convertComponentsToMutationVariables,
 } from '../components/GraphqlQueryRenderer/GraphqlQueryRenderer';
-import { FormComponent } from '../questionTypes';
+import { mapMutationVariableToFormComponent } from '../mapping';
+import { FormComponent, MutationVariableInput } from '../questionTypes';
 
 import { FormBuilder } from './FormBuilder/FormBuilder';
 
 interface QuestionnaireBuilderProps {
   hirerId?: string;
+  graphqlInputVariables?: MutationVariableInput;
```

Comment:
1. It's confusing that there's a hirer ID nested in `graphqlInputVariables` that is ignored in favour of the separate `hirerId` prop. It might make sense to drop `hirerId`.
1. There no plural _variables_ here; how about just `graphqlInput`?

Reply:
1. I think Ryanair gets the `hirerId` from `user`. Since `graphqlInput` is optional, there might be no hirerId? or should there be a check to see if hirerId is the same in the `graphqlInput`?
2. sure